cupyx/jit/_cuda_typerules.py

```diff
@@ -140,7 +141,7 @@ def get_ctype_from_scalar(mode, x):
     raise NotImplementedError(f'{x} is not scalar object.')
 
 
-_typechars = '?bBhHiIlLefdFD'
+_typechars = '?bBhHiIlLqQefdFD'
```

Comment:
@asi1024 Could you confirm this is OK? I seem to remember we removed `qQ` for good reason (perhaps for Windows)?

Reply:
Maybe [this fix](https://github.com/cupy/cupy/pull/4794/files#diff-178317a5acc0687f9d0bc99003038b5b5b754a98e4fbccb8024761fc81d88a6fR122) resolved most CuPy JIT issues on Windows!